### PR TITLE
Run vcdiff_test in make test.

### LINKED ISSUE #26: vcdiff_test.sh specifies potentially incorrect shell location, test fails
Original [issue 26](https://code.google.com/p/open-vcdiff/issues/detail?id=26) created by greenreaper on 2010-03-18T14:54:43.000Z:

<b>What steps will reproduce the problem?</b>
1. Download source package and extract with tar xvf
2. Run ./configure &amp;&amp; make &amp;&amp; make check

<b>What is the expected output? What do you see instead?</b>

I expect all the tests to pass. Instead, the last test involving
src/vcdiff_test.sh did not pass.

The error code given was:
./src/vcdiff_test.sh: not found
FAIL: src/vcdiff_test.sh

<b>What version of the product are you using? On what operating system?</b>

Version 0.7 on FreeBSD 7.1-RELEASE-p8 in an OS-based jail.

<b>Please provide any additional information below.</b>

On this system, bash is located in /usr/local/bin/bash. The first line of
the .sh file src/vcdiff_test.sh specifies /bin/bash

When unpacked, the source file was not writable. If set writable, and the
shebang line line was removed, or changed to specify /usr/local/bin/bash,
the test worked.

### PR DESCRIPTION
**Pull request by bacek**
Also change to run it with /bin/sh which is perfectly fine in this case.
Fixes #26

### CODE REVIEW

**Comment by bacek** [comment]
I'm not sure how to block running this test on Windows.

**Comment by bacek** [comment]
It should be good to go. Theoretically. I don't have Windows with build environment set up to verify it.

**Comment by bacek** [comment]
Up

File: src/vcdiff_test.bat [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+fc /b %TARGET_FILE% %OUTPUT_TARGET_FILE% ^
+    || ( echo Decoded target does not match original ^
+         &&exit /b 1 )
+echo Test 3 ok
+
+del %DELTA_FILE%
+del %OUTPUT_TARGET_FILE%
+
+rem open-vcdiff Issue 7
+rem (https://github.com/google/open-vcdiff/issues/detail?id=7)
```
Comment (Steelskin) [inline src/vcdiff_test.bat]: Wrong link: https://github.com/google/open-vcdiff/issues/7

File: src/vcdiff_test.bat [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+fc /b %TARGET_FILE% %OUTPUT_TARGET_FILE% ^
+    || ( echo Decoded target does not match original ^
+         &&exit /b 1 )
+echo Test 13 ok
+
+del %DELTA_FILE%
+del %OUTPUT_TARGET_FILE%
+
+rem open-vcdiff Issue 6
+rem (https://github.com/google/open-vcdiff/issues/detail?id=6)
```
Comment (Steelskin) [inline src/vcdiff_test.bat]: same here.

File: src/vcdiff_test.bat [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+
+%VCDIFF% %VCD_OPTIONS% ^
+         test -dictionary %DICTIONARY_FILE% ^
+              -delta %DELTA_FILE% ^
+    && ( echo vdiff test without target option should fail, but succeeded ^
+         &&exit /b 1 )
+echo Test 33 ok
+
+rem open-vcdiff Issue 8
+rem (https://github.com/google/open-vcdiff/issues/detail?id=8)
```
Comment (Steelskin) [inline src/vcdiff_test.bat]: and here

**Comment by Steelskin** [comment]
Thanks! I added a couple comments about the links to the issues.

**Comment by bacek** [comment]
Fixed.

**Pull request merged**